### PR TITLE
add australian server

### PR DESCRIPTION
**Pull request by Worf1337**

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -56,5 +56,13 @@
             "name": "Japan 1",
             "url": "https://jp1.watchtube.app/"
         }
```
Comment (BBaoVanC) [inline index.json:58]: Needs comma

```suggestion
        },
```

File: index.json [hunk 2]
```diff
@@ -56,5 +56,13 @@
...
+        {
+            "region": "oceania",
+            "regionDisplay": "Oceania",
+            "country": "au",
+            "countryDisplay": "Australia",
+            "name": "Australia 1",
+            "url": "https://au1.watchtube.app/",
```
Comment (BBaoVanC) [inline index.json:65]: Remove comma

```suggestion
            "url": "https://au1.watchtube.app/"
```

**Review by BBaoVanC — APPROVED** [review]
Just needs DNS record to be added

**Comment by llsc12** [comment]
Agreed. I'll merge when dns is set up and I test it

**Pull request closed without merging**